tools/monitoring/mysql/error_logger.go

```diff
@@ -18,50 +18,38 @@ package mysql
 
 import (
 	"database/sql"
+	"fmt"
 	"time"
 
 	"github.com/knative/test-infra/shared/mysql"
-	"github.com/knative/test-infra/tools/monitoring/config"
-	"github.com/knative/test-infra/tools/monitoring/log_parser"
 )
 
-const (
-	logInsertStmt = `
-	INSERT INTO ErrorLogs (
-		ErrorPattern, ErrorMsg, JobName, PRNumber, BuildLogURL, TimeStamp
-		) VALUES (?,?,?,?,?,?)`
-)
+// MonitoringDatabase holds an active database connection created in `config`
+type MonitoringDatabase struct {
```

Comment:
DB should be fine. It's under the monitoring folder already and we don't want to mix the use of sql.DB and this struct in a same file.

Reply:
Thanks for pointing it out. Updated.